notebook/static/notebook/js/menubar.js

```diff
@@ -130,6 +130,158 @@ define([
         this._new_window(url);
     };
 
+    MenuBar.prototype._nbconvert_upload_conf = function(download) {
+      var body = $("<div>");
+      var notebook_path = utils.encode_uri_components(
+        this.notebook.notebook_path
+      );
+
+      const build_json_for_post = function(notebook, config, format) {
+        const json_to_pass = {
```

Comment:
If I had to guess, these are causing the test suite to fail since it's ES3. You'll need to switch to `var` here.

Reply:
If that's not what's causing the issue, I'm out of options as I don't know how to diagnose the old suite. It just fails flatly if there is a parsing issue within phantom.